web-components/src/[sandbox]/examples/date-time-picker.ts

```diff
@@ -23,7 +23,6 @@ export const dateTimePickerTemplate = html`
   <h3>date-time-picker with value</h3>
   <md-date-time-picker
     value="2021-02-14T12:00:00-08:00"
-    @date-time-change=${() => console.log("fart")}
```

Comment:
lol thanks for finding that someone did this and removing

Reply:
haha, true.Thanks for catching it.